test/test.go

```diff
@@ -0,0 +1,34 @@
+package test
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/nimblehq/mark-ic/bootstrap"
```

Comment:
You might need to use the right repo URL for importing

```suggestion
	"https://github.com/markgravity/golang-ic/bootstrap"
```

Reply:
@suho it's weird the current import is correct 🤔 
![CleanShot 2023-08-15 at 10 22 11](https://github.com/markgravity/golang-ic/assets/17875522/a24a7fe4-1f64-41e1-90c3-af8d06eed6d4)

Reply:
@markgravity This is defined in the `go.mod` file

Reply:
@suho Gotcha, updated at [437555d](https://github.com/markgravity/golang-ic/pull/18/commits/437555d9ac6a64630506bc5bc13272c8874e7cff)